src/serialization.cpp

```diff
@@ -152,6 +152,18 @@ size_t JsonWriteRichPresenceObj(char* dest,
                     WriteOptionalString(writer, "join", presence->joinSecret);
                     WriteOptionalString(writer, "spectate", presence->spectateSecret);
                 }
+                else 
```

Comment:
Buttons can't be at the same time as match/join/spectate?

Reply:
Yeah, u must to have choose between one of them.